src/demo/HomePageA/components/SoftKey/SoftKey.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback, useEffect } from 'react';
```

Comment:
Can you please point me to any new change you added. If not, remove it & use it directly from kaiui components.

Reply:
Closing this as some changes are made